source/reduce/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2018 Google Inc.
```

Comment:
These should be Google LLC.

Reply:
Thanks.  (Note that there seem to be quite a few instances of this in the existing code.)

Reply:
Fixed (the instances introduced in this PR).